src/store-api/src/region_request.rs

```diff
@@ -397,8 +398,12 @@ impl RegionCreateRequest {
     }
 }
 
-#[derive(Debug, Clone, Default)]
-pub struct RegionDropRequest {}
+#[derive(Debug, Clone)]
+pub struct RegionDropRequest {
+    /// fast drop database path for the shortcut that do not need to delete logical
+    /// columns first more to check issue #4974 and #5561
+    pub fast_drop_path: bool,
+}
```

Comment:
Any idea about defining the field? @evenyag @fengjiachun

Reply:
will change the name after discuss 
IMO force_drop_all_logical_tables is kind of better.
but as mention by @fengjiachun 
> How about using fast_drop_path? I believe we might also offer the ability to fast drop physical tables in the future, so it's not just for dropping databases.

Reply:
`force_drop_all_logical_tables` +1